### ISSUE TITLE
Vortex issues

### ISSUE THREAD

**Issue by solacelost** [sonic2kk/steamtinkerlaunch#57]
Fedora 32 system, fully updated. Steam installed from `rpmfusion` repo, Proton installations all local to user.

Description of steps I've taken:
* Initially downloaded `stl`, dropped in `~/.local/bin/stl`
* Set up a Steam collection named `VORTEX`
* Added Skyrim: SE to the collection, and clicked Play. After shader cacheing, nothing happened.
* Checked command line arguments out. Ran `stl vortex install`
* Several errors about no `proton` in $PATH. Ran `stl vortex start`. Error about Vortex not being installed into prefix.
* Symlinked `proton` binary from Steam Proton 5.0 installation into $PATH at `~/.local/bin/proton`
* `stl vortex install` and `stl vortex start` similar behavior, although install no longer complained about `proton` being missing, it showed some wintricks errors I didn't think to be important.
* Completely deleted `~/.config/stl` and reran `stl vortex install` from command line. Installer appears to be downloaded, not installed.
* Game continues to not launch.
* Rerunning at this point `stl vortex install` shows  winetricks error as pasted here:

```
$ stl vortex install
Executing mkdir -p /home/james/.config/stl/vortex
------------------------------------------------------
You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
Using winetricks 20200412 - sha256sum: 6d2b4ec442f88dca514115d6d2c35c32d8923e15e8ee81b13c2b7e62c0d0bccd with wine-5.15 (Staging) and WINEARCH=win64
Executing w_do_call dotnet48
Executing mkdir -p /home/james/.config/stl/vortex
------------------------------------------------------
You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
Executing load_dotnet48 
------------------------------------------------------
This package (dotnet48) may not fully work on a 64-bit installation. 32-bit prefixes may work better.
------------------------------------------------------
Executing w_do_call remove_mono
Executing mkdir -p /home/james/.config/stl/vortex
------------------------------------------------------
You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
Executing load_remove_mono 
------------------------------------------------------
Mono does not appear to be installed.
------------------------------------------------------
Executing w_do_call dotnet40
Executing mkdir -p /home/james/.config/stl/vortex
------------------------------------------------------
You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
Executing load_dotnet40 
------------------------------------------------------
This package (dotnet40) may not fully work on a 64-bit installation. 32-bit prefixes may work better.
------------------------------------------------------
------------------------------------------------------
dotnet40 does not yet fully work or install on wine.  Caveat emptor.
------------------------------------------------------
Current Wine does not have Wine bug 42701, so not applying workaround
Executing w_do_call remove_mono
Executing mkdir -p /home/james/.config/stl/vortex
------------------------------------------------------
You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
Executing load_remove_mono 
------------------------------------------------------
Mono does not appear to be installed.
------------------------------------------------------
Executing w_do_call winxp
Executing mkdir -p /home/james/.config/stl/vortex
------------------------------------------------------
You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
Executing load_winxp 
The operation completed successfully
Setting Windows version to winxp
Executing /usr/bin/wine regedit /S C:\windows\Temp\set-winver.reg
Executing /usr/bin/wine64 regedit /S C:\windows\Temp\set-winver.reg
------------------------------------------------------
Running /usr/bin/wineserver -w. This will hang until all wine processes in prefix=/home/james/.config/stl/vortex/wineprefix terminate
------------------------------------------------------
Executing cd /home/james/.cache/winetricks/dotnet40
Executing /usr/bin/wine dotNetFx40_Full_x86_x64.exe /q /c:install.exe /q
WARNING: Experimental compiler backend enabled. Here be dragons! Incorrect rendering, GPU hangs and/or resets are likely
------------------------------------------------------
Note: command /usr/bin/wine dotNetFx40_Full_x86_x64.exe /q /c:install.exe /q returned status 123. Aborting.
------------------------------------------------------
```

I made a few attempts at manually installing into the prefix without fiddling with winecfg options too hard, no love.

There are a lot of variables set by your script, and I feel like attempting to recreate them in order to debug this further will be troublesome. I'm just not going to use it for now. Let me know where I'm messing up, what additional debugging/versioning info you'd like, and what I can do to help.

Thanks! :)

**Comment by solacelost** [comment]
_sigh_ nevermind :)
https://bugs.winehq.org/show_bug.cgi?id=49532

**Comment by solacelost** [comment]
fwiw:
`WINE=~/.steam/steam/steamapps/common/Proton\ 5.0/dist/bin/wine WINEPREFIX=~/.config/stl/vortex/wineprefix winetricks dotnet48` got .NET 4.8 installed

`WINE=~/.steam/steam/steamapps/common/Proton\ 5.0/dist/bin/wine stl vortex install` got other dependencies installed

`WINEPREFIX=~/.config/stl/vortex/wineprefix ~/.steam/steam/steamapps/common/Proton\ 5.0/dist/bin/wine ~/.config/stl/downloads/Vortex/vortex-setup-1.2.20.exe` got Vortex installed itself and now, finally:

`stl vortex start` works to start Vortex. :)

**Comment by frostworx** [comment]
I already closed vortex/dotnet issues.

When you look into the code you see that WINE is set to VORTEXWINE _(function wineVortexRun)_.
If VORTEXWINE is not configured in vortex.conf at all it falls back to "$GWINE" _(function initVortexStuff)_
GWINE in turn used currently used PROTON wine, if that is not found it tries to pick the last used PROTON wine from file "$LASTRUN" and if that doesn't exist it falls back to system wine (which apparently is too new on your system)

tl;dr: It works. If it doesn't it should be enough to configure VORTEXWINE.
If not (still possible that I broke something during updates) it might help to check the logs.
